docs/api/MQTT.md

```diff
@@ -113,8 +113,17 @@ Unsubscribe from a topic.
 
 Close the client.
 
+### client.disconnect([err, [, callback]])
```

Comment:
What's the difference between `client.end` and `client.disconnect`?

Reply:
they both do the same thing, the MQTT.js use `end` and we use `disconnect` before, so keep `disconnect` and add `end` to make compatible with `MQTT.js` and old apps